docs/health/health_design/design.rst

```diff
@@ -34,27 +90,29 @@ Here is the pyATS Health Check yaml. It's almost same with `Blitz`! There are a
                 processor: both
                 # `function` can be found from Genie Feature Browser
                 # Please find the link to the page from bottom of this section
-                function: get_platform_cpu_load
+                function: health_cpu
                 arguments:
                   command: show processes cpu
                   processes: ['BGP I/O']
         - memory:
             - api:
                 device: uut
                 processor: post
-                function: get_platform_memory_usage
+                function: health_memory
```

Comment:
why is there a health specific api?

Reply:
good catch! yes. dedicated health API starting with `health_`. but it returns more data for health. but it doesn't mean user cannot use generic APIs. user still can use generic genie APIs as well if they want.